src/app/address/controllers/summary/nonUKConfirm.js

```diff
@@ -0,0 +1,54 @@
+const { PACKAGE_NAME } = require("../../../../lib/config");
+const BaseController = require("hmpo-form-wizard").Controller;
+const logger = require("hmpo-logger").get(PACKAGE_NAME);
+
+const {
+  generateHTMLofNonUKAddress,
+} = require("../../../../presenters/addressPresenter");
+
+const saveAddressess = require("./utils/saveAddresses");
+
+const CHANGE_CURRENT_HREF = "/enter-non-UK-address";
```

Comment:
The "?edit=true" stuff is for analytics we probably want it here too ([docs](https://www.npmjs.com/package/@govuk-one-login/frontend-analytics#user-content-form-change-tracker))

Also the /edit is something to do with [hmpo-form-wizard](https://github.com/HMPO/hmpo-form-wizard) but I can't really figure out what exactly it does so I guess it's safe to remove?

Reply:
Ah nice, thanks for the link also. I've added `?edit=true`

I've had a play with `/edit` and `editable` in the routes and it does not seem to make any difference for our use case, so I'm unsure also 🤷